test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -44,8 +39,10 @@ func TestSchedule100Node3KPods(t *testing.T) {
 	if testing.Short() {
 		t.Skip("Skipping because we want to run short tests")
 	}
-
-	config := defaultSchedulerBenchmarkConfig(100, 3000)
+	config := baseConfig()
```

Comment:
Instead of removing `default...` function maybe just re-implement it with those 3 lines?